speaking-portal-project/build/index.d.ts

```diff
@@ -1 +0,0 @@
-export {};
```

Comment:
I think potentially this was pushed before the gitignore was updated causing it to be included - I wonder if we could remove this?